AppCenter/AppCenter/Internals/Model/CommonSchema/MSCommonSchemaLog.m

```diff
@@ -5,6 +5,15 @@
 
 @implementation MSCommonSchemaLog
 
+static NSString *const kMSTypeCommonSchema = @"commonSchema";
+
+- (instancetype)init {
+  if ((self = [super init])) {
+    self.type = kMSTypeCommonSchema;
```

Comment:
This is a debugging purpose. It looks weird and is hard to understand when a log message says "Sending a log type: (null) ... ".
We don't send a type for common schema so it isn't used at all. Maybe I can add a comment there?

Reply:
To me type should be "event" for instance not common schema. So the meaning of the type property would be the same between AC logs and CS logs.

Reply:
It is just for debugging purpose. If we use event for the type, it would be confusing. If you have a concern about this one than I can revert the change. What do you think?

Reply:
I'd prefer to revert the change but setting the type to event in the MSEventLog.m `toCommonSchemaLogForTargetToken:` seems an easy change.

Reply:
Will revert since this one is not in scope for the PR.